uswds/uswds
# USWDS: Replaced deprecated xlink:href code with recommended use of href

# Summary


All `xlink:href` instances have be replaced with `href` within the repo in order to account for the deprecation of `xlink:href`.

## Breaking change

_Indicate if this update is a breaking change with **one** of the following statements:_
This is not a breaking change.

## Related issue

Closes #6088 


## Related pull requests

[USWDS-Site: Replaced deprecated xlink:href code with recommended use of href #2926](https://github.com/uswds/uswds-site/pull/2926)

## Preview link

Preview link:
[USWDS Storybook Icon](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/rc-xlink-deprecated/?path=/story/components-icon-list--default)

## Problem statement

The USWDS repository currently contains numerous instances of xlink:href, which is a deprecated attribute that may cause compatibility issues/errors.

## Solution

The USWDSrepository has been updated to use the `href` attribute instead of the deprecated `xlink:href` attribute.

## Markup Change
Users should update their `xlink:href` references to href.

## Testing and review

1. Open up local version of USWDS Storybook or preview link
2. Ensure icons are working as expected
3. Using dev tools, ensure the `<use>` no longer uses `xlink:href` and uses `href` instead when inspecting an icon
